_layouts/doc_page.html

```diff
@@ -52,7 +52,12 @@
 
     <script>
     $(function() {
-      $(".toc").load("/docs/{{ version }}/toc.html");
+      $toc = $(".toc");
+      $toc.load("/docs/{{ version }}/toc.html", function () {
+        $toc.find('a').each(function(idx, el) {
+          el.href = el.href.replace('replace-with-version', '{{ version }}');
```

Comment:
any reason we need to use JS for this? If we make the toc use a blank template that just defines the version variable we can just use the liquid template variable directly.